dapr/actor/runtime/_reminder_data.py

```diff
@@ -72,7 +72,7 @@ def as_dict(self) -> Dict[str, Any]:
         if self._state is not None:
             encoded_state = base64.b64encode(self._state)
         return {
-            'name': self._name,
+            'reminderName': self._reminder_name,
```

Comment:
@yaron2 @youngbupark Similar to this should `timerName` be returned in `as_dict` call for the timer class seen below?
See line 79 in `_timer_data.py` file.